advocacy_docs/partner_docs/VeritasNetBackupforPostgreSQL/05-UsingVeritasNetBackupForPostgreSQL.mdx

```diff
@@ -1,51 +1,49 @@
 ---
-title: 'Using Veritas NetBackup for PostgreSQL'
+title: 'Using'
 description: 'Walkthroughs of multiple Veritas NetBackup for PostgreSQL usage scenarios'
 ---
 
 Common backup/restore operations for PostgreSQL databases using Veritas NetBackup for PostgreSQL are:
-- Performing backups - takes backup of the database and stores it in a predetermined location.
-- Querying backups - lists available database backups.
-- Performing restores - restores the database from a backup previously taken:
-    - Local restore - database files are restored to the same host as the client.
-    - Redirected restore - database files are restored to a different host than the client.
+- Performing backups &mdash; Takes backup of the database and stores it in a predetermined location.
+- Querying backups &mdash; Lists available database backups.
+- Performing restores &mdash; Restores the database from a backup previously taken:
+    - Local restore &mdash; Database files are restored to the same host as the client.
+    - Redirected restore &mdash; Database files are restored to a host different from the client.
 
 
 
-### Performing Backups
+## Performing backups
 
-To take a backup of the database, enter the command on the Veritas NetBackup Client:
+To take a backup of the database, enter the command on the Veritas NetBackup client:
 
 `/usr/NBPostgreSQLAgent/nbpgsql -o backup`
 
 
-### Querying Backups
+## Querying backups
 
-To list available database backups, enter the command on the Veritas NetBackup Client:
+To list available database backups, enter the command on the Veritas NetBackup client:
 
 `/usr/NBPostgreSQLAgent/nbpgsql -o query`
-<p align="center">
-    <img  src="Images/BackupQuery.png">
-</p>
 
-### Performing Restores
+![Query Backup](Images/BackupQuery.png)
 
-Database restores can be performed in the following two scenarios:
+## Performing restores
 
-- Local Restore.
-- Redirected Restore.
+You can perform database restores in the following two scenarios:
 
+- Local restore
+- Redirected restore
 
-#### Local Restore
 
-In this scenario, the database files are restored on the original (source) database server host (default 
-option). 
+### Local restore
 
-To perform a local restore, use the steps below: 
+In this scenario, the database files are restored on the original (source) database server host. This is the default option. 
+
+To perform a local restore: 
 
 1. Stop the database server.
 
-2. Create the target directory to store the database files to be used for the restore operation.
+2. Create the target directory to store the database files to use for the restore operation.
 
 3. Determine the database backup id you want to use for the restore by querying available backups:
```

Comment:
Should id be ID?

Reply:
I never change that. It's basically all over the place but I would say for the most part they use lower case. I would leave it.